css/css-fonts/parsing/font-variant-valid.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Fonts Level 3: parsing font-variant with valid values</title>
```

Comment:
Please  remove the font-variant-alternates values from this test (moved from Fonts 3 to Fonts 4)